backend/openapi.yaml

```diff
@@ -3921,6 +3922,9 @@ components:
           format: date
           nullable: true
           readOnly: true
+        on_grace_period:
+          description: Do any of the conditions of sale related to this apartment have a grace period set?
+          type: boolean
```

Comment:
If we wanted to describe if this apartment is currently on grace period, the field should be renamed to such that it hint that outputs a boolean value: `is_on_grace_period`.

But now this field is a bit confusingly named i think,  as it doesn't tell if the apartment is currently on grace period, it tells does it have any grace period currently given, and so we should rename it to e.g. `has_grace_period_set` or something that does describe what it actually is.

Reply:
I can see how it might be confusing, changed it to just `has_grace_period`.